pegasus/emails/hoc_signup_2016_receipt.md

```diff
@@ -0,0 +1,40 @@
+---
+from: "Hadi Partovi (Code.org) <hadi_partovi@code.org>"
+subject: "Thanks for signing up to host an Hour of Code!"
+litmus_tracking_id: "5g5lyi1a"
```

Comment:
Yeah, I asked somebody about it last time. Maybe Rick or Laurel. They said
use the same ID.

On Wed, Jun 22, 2016 at 2:53 PM, Brendan Reville <notifications@github.com>
wrote:

> In pegasus/emails/hoc_signup_2016_receipt.md
> <https://github.com/code-dot-org/code-dot-org/pull/9124#discussion_r68142452>
> :
>
> > @@ -0,0 +1,40 @@
> > +---
> > +from: "Hadi Partovi (Code.org) <hadi_partovi@code.org>"
> > +subject: "Thanks for signing up to host an Hour of Code!"
> > +litmus_tracking_id: "5g5lyi1a"
>
> presume this ID is correct for 2016?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/9124/files/eefb579935fa25e0dab69cdad4b2aca692ac7217#r68142452>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB1OEnjz6WXtTZTqxGaFmouqBKeSikmzks5qOa7ZgaJpZM4I8Qd3>
> .
>



-- 
Tanya Parker
Product Manager, Code.org
+1 (562) 318-4137